ui/src/app/applications/components/application-parameters/application-parameters.tsx

```diff
@@ -286,7 +286,7 @@ export const ApplicationParameters = (props: {
     } else if (props.details.type === 'Plugin') {
         attributes.push({
             title: 'NAME',
-            view: <div style={{marginTop: 15, marginBottom: 5}}>{ValueEditor(app.spec.source.plugin && app.spec.source.plugin.name, null)}</div>,
+            view: <div style={{marginTop: 15, marginBottom: 5}}>{ValueEditor(app.spec.source?.plugin && app.spec.source.plugin.name, null)}</div>,
```

Comment:
I think this can be shortened as well 

```suggestion
            view: <div style={{marginTop: 15, marginBottom: 5}}>{ValueEditor(app.spec.source?.plugin?.name, null)}</div>,
```

Reply:
Indeed. Simplified.